students/uss_tomasz/lesson_13_classes/13_4/selective_shallow_compare.py

```diff
@@ -0,0 +1,29 @@
+from pprint import pprint
+
+
+def selective_shallow_compare(object1, object2, attributes_list):
+    """Function that for given 2 objects and list of attribute names checks if objects' attributes are equal."""
+    for attribute in attributes_list:
+        if getattr(object1, attribute, 'not found') != getattr(object2, attribute, 'not found'):
```

Comment:
Solutions based on artificial string literals are error prone. Check this example where in fact one object has the attribute, whilst the other doesn't:
```
a = {}
b = {'attr': 'not found'}

print(getattr(a, 'attr', 'not found') == getattr(b, 'attr', 'not found'))
```

You should check first if both have the attribute, then check if both are the same